packages/dds/tree/src/test/shared-tree/sharedTree.spec.ts

```diff
@@ -106,5 +106,45 @@ describe("SharedTree", () => {
                 tree2.forest.forgetAnchor(destination);
             }
         });
+
+        it("can insert multiple nodes", async () => {
```

Comment:
Is this test supposed to be part of this PR?

Reply:
I was on the fence about it. It's how I ran into the issue (the test fails without this new code) so I figured it doesn't hurt to keep it.

Reply:
Happy to take it out if you find it distasteful.